controllers/storagecluster/reconcile.go

```diff
@@ -172,6 +172,13 @@ func (r *StorageClusterReconciler) Reconcile(ctx context.Context, request reconc
 		return reconcile.Result{}, err
 	}
 
+	// Ensure that verbose logging or debugging is enabled when RBD mirroring is enabled
+	// TODO: This is a temporary arrangement, this is to be removed when RDR goes to GA
+	result, err = r.ensureRbdMirrorDebugLogging(sc)
```

Comment:
I don't think this is the correct place to call this method, We should move this to the reconcilePhases